code/datums/gamemodes/combat_patrol.dm

```diff
@@ -0,0 +1,189 @@
+/datum/game_mode/combat_patrol
+	name = "Combat patrol"
```

Comment:
```suggestion
	name = "Combat Patrol"
```

Reply:
This actually caused a funny bug... gotta double check if I use the lower case elsewhere first.

Reply:
done.